esy.json

```diff
@@ -4,15 +4,12 @@
   "license": "MIT",
   "version": "3.3.3",
   "dependencies": {
-    "@esy-ocaml/substs": "^0.0.1",
-    "@esy-ocaml/esy-installer": "^0.0.0",
-    "@opam/ocamlfind": "",
+    "@opam/ocamlfind": "*",
     "@opam/menhir": " >= 20170418.0.0",
     "@opam/utop": " >= 1.17.0",
-    "@opam/merlin-extend": " >= 0.3.0",
+    "@opam/merlin-extend": " >= 0.3",
     "@opam/result": "*",
     "@opam/ocaml-migrate-parsetree": "*",
-    "@opam/ocamlbuild": "*",
     "@opam/dune": "*"
   },
   "peerDependencies": {
```

Comment:
Let's move `peerDependencies` to `dependencies`, `peerDependencies` are an artefact of npm and are not needed with esy as esy has proper constraint solver.